modules/standard/Path.chpl

```diff
@@ -260,5 +260,32 @@ proc file.realPath(): string throws {
       }
     }
    return result;
- }  
+ }
+ 
+/* Determines whether the path specified is an absolute path in Unix
+   environment and returns error if it is not in Unix environment
+
+   :arg name: the path to be checked.
+   :type name: `string`
+
+   :return: `true` if `name` is an absolute path, `false` otherwise
+   :rtype: `bool`
+*/
+
+  proc isAbsPath(name: string): bool {
+     if (CHPL_TARGET_PLATFORM == 'linux64' || CHPL_TARGET_PLATFORM == 'linux32' || CHPL_TARGET_PLATFORM == 'darwin') {
+        if name.isEmptyString() {
+           return false;
+        }
+        const len: int = name.length;
+        var str: string = name[1];
+        if (str == '/') {
+          return true;
+        }
+        else 
+          return false;
+     }
+     else
+     writeln("Error : Target platform should have Unix like environment");
```

Comment:
The writeln should be indented under the else, especially since there are no curly braces to make the dependence obvious.

Reply:
Actually, the more I think about it, the more I wonder if throwing an Error subclass would be useful here (see [this document](https://chapel-lang.org/docs/latest/technotes/errorHandling.html) for general error handling in Chapel)

Reply:
@psahabu - is there a handy list somewhere of all the Error subtypes, that @victor-ludorum could use?

Reply:
We've created a SystemError subtype hierarchy, which can be found here: https://chapel-lang.org/docs/1.16/modules/standard/SysError.html?highlight=syserror#SysError.SystemError

We're open to creating other subtype hierarchies, though it's not an area of focus right now.

Reply:
Would making another subtype of SystemError be reasonable for this error?  The error is basically "we don't support this in your environment yet", and it doesn't look like anything is already present that would work for that.

Reply:
@mppf here's the discussion